### ISSUE TITLE
Error in tutorial

### ISSUE THREAD

**Issue by hongbo-miao** [Urigo/meteor-angular2.0-socially#38]
If use any new version (newer than version 0.7) of urigo:angular2-meteor, it will show a blank page and errors in console:
![screenshot](https://cloud.githubusercontent.com/assets/3375461/12627161/43676232-c50a-11e5-82cd-e798d3561e8b.png)

Based on the content, in the [tutorial](http://www.angular-meteor.com/tutorials/socially/angular2/bootstrapping) from the step 0.5 to step 3.5, `import {bootstrap} from 'angular2/bootstrap';`  should change to `import {bootstrap} from 'angular2/platform/browser';`

After step 3.6, it is correct, which is `import {bootstrap} from 'angular2-meteor';`.

**Comment by justdoit176** [comment]
Duplicate of #37
